yb-voyager/src/metadb/migrationStatus.go

```diff
@@ -15,15 +17,27 @@ type MigrationStatusRecord struct {
 	TargetDBConf                *tgtdb.TargetConf
 	FallForwardDBConf           *tgtdb.TargetConf
 	TableListExportedFromSource []string
+	Triggers                    []string
```

Comment:
Let's have named boolean flag member variables to indicate the "trigger" and "status" flags:
```
CutoverRequested bool
CutoverProcessedBySourceExporter bool
CutoverProcessedByTargetImporter bool
FallForwardSwitchRequested bool
FallForwardSwitchProcessedByTargetExporter bool
FallForwardSwitchProcessedByFFImporter bool
```